pkg/arvo/lib/ph/io.hoon

```diff
@@ -186,27 +138,11 @@
   loop
 ::
 ++  init-ship
-  |=  =ship
-  =/  m  (strand ,~)
-  ^-  form:m
-  ~&  >  "starting {<ship>}"
-  ;<  ~  bind:m  (send-events (init:util ship `*dawn-event:jael))
-  (check-ship-booted ship)
-::
-++  real-ship
-  |=  [=tid:spider =ship]
-  ~&  >  "booting real {<ship>}"
-  =/  m  (strand ,~)
-  =/  =vase  !>([tid %azimuth-command !>([%create-ship ship])])
-  ;<  ~  bind:m  (poke-our %spider %spider-input vase)
-  (check-ship-booted ship)
-::
-++  raw-ship
-  |=  [=ship keys=(unit dawn-event:jael)]
```

Comment:
Seems like it'd be worth keeping around the `+fake-ship` / `+real-ship` api.

Reply:
It's still around, just by a different name. `++init-ship` takes a flag indicating whether it's a fakeship. It's worth noting that some of these functions were never updated to use the new azimuth model, where the state is persistent in the aqua agent.